src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
+    if (animal.hidden === false && animal instanceof Animal) {
+      animal.health -= 50;
+    }
+
+    if (animal.health <= 0) {
+    // eslint-disable-next-line
+      const beastIndex = Animal.alive.findIndex(el => el.health <= 0 && el.name === animal.name);
```

Comment:
you already check if animal health is less 0 in condition above
try this one or something like that
```suggestion
      const beastIndex = Animal.alive.findIndex(el => animal);
```